src/MuTalk-Model/MTAnalysis.class.st

```diff
@@ -53,38 +53,44 @@ MTAnalysis >> coverageAnalysisResult: anObject [
 
 { #category : 'accessing - defaults' }
 MTAnalysis >> defaultBudget [
+	"Since tests often take little time to run, we multiply it by 30"
 
-	^ MTFreeBudget new
+	^ MTTimeBudget for: self totalTestsTime * 30
 ]
 
 { #category : 'accessing - defaults' }
 MTAnalysis >> defaultLogger [
 
-	^ MTNullLogger new
+	^ MTTranscriptLogger new
 ]
 
 { #category : 'accessing - defaults' }
 MTAnalysis >> defaultMutantGenerationStrategy [
 
-	^ MTAllMutantGenerationStrategy new
+	^ MTAllMutantGenerationStrategy withRandomOperatorMutantSelection
 ]
 
 { #category : 'accessing - defaults' }
 MTAnalysis >> defaultOperators [
 
-	^ MTAbstractMutantOperator contents
+	^ MTAbstractMutantOperator contentsAll
```

Comment:
We should review the operators, I don't think it will make sense to have both contents and contentsAll